packages/graphql_flutter/pubspec.yaml

```diff
@@ -3,7 +3,8 @@ description: A GraphQL client for Flutter, bringing all the features from a mode
 version: 5.0.1
 homepage: https://github.com/zino-app/graphql-flutter/tree/master/packages/graphql_flutter
 dependencies:
-  graphql: ^5.0.1-beta.5
+  graphql:
+    path: ../graphql
```

Comment:
Let's change this back once we've released the other PRs.

Reply:
We need to update our release system in some way. However, i bump the new version and reverted this change.

Thanks for this.